models/nequip/test_nequip_discovery.py

```
@@ -0,0 +1,163 @@
+from ase.filters import ExpCellFilter, FrechetCellFilter
+from nequip.ase import NequIPCalculator
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatviz.enums import Key
+from tqdm import tqdm
+
+from matbench_discovery import timestamp
+from matbench_discovery.data import DataFiles, as_dict_handler, ase_atoms_from_zip
+from matbench_discovery.enums import Task
+
+with contextlib.suppress(ImportError):
+    # OpenEquivariance/CuEquivariance libraries need to be loaded to allow their use in ASE calculators, if model was compiled with these accelerations
```

Ruff (0.12.2)

Line too long (151 > 88)

(E501)